hadoop-common-project/hadoop-common/src/main/resources/core-default.xml

```diff
@@ -1093,12 +1101,48 @@
 </property>
 
 <property>
-  <name>fs.s3a.fast.upload</name>
+  <name>fs.s3a.block.output</name>
```

Comment:
Is this revision missing the changes to restore/un-deprecate `fs.s3a.fast.upload`?